app/_components/organisms/Navigation.tsx

```diff
@@ -21,18 +17,18 @@ const Navigation: React.FC<NavigationProps> = ({ type }) => {
   const [isSearchModalOpen, setIsSearchModalOpen] = useState(false);
 
   const shopItems: MenuProps["items"] = [
-    "Shop All",
-    "Prints",
-    "Books",
-    "Stickers",
-    "Keychains",
-    "Standee",
-    "Tumbler / Mugs",
-    "Tshirts / Hoodies",
-    "Paocat Merch",
-  ].map((label, index) => ({
+    { label: "Shop All", slug: "all" },
+    { label: "Prints", slug: "prints" },
+    { label: "Books", slug: "books" },
+    { label: "Stickers", slug: "stickers" },
+    { label: "Keychains", slug: "keychains" },
+    { label: "Standee", slug: "standee" },
+    { label: "Tumbler / Mugs", slug: "tumbler_mugs" },
+    { label: "Tshirts / Hoodies", slug: "tshirts_hoodies" },
+    { label: "Paocat Merch", slug: "paocat" },
```

Comment:
there should be a utility using lodash to transform string to snake_case. maybe instead of using an object we can still have an array and then convert it to snake_case for slug

Reply:
installed lodash, used _.snakeCase to simplify the array